### PR TITLE
feat(cmd): version cmd prints short semver info [skip-e2e]

### LINKED ISSUE #390: Print just version of the ike binary
##### What can be improved

At the moment we print lots of details about the binary build when asking for `version`:

```
╰─ ike version
Ike Version: v0.0.2-next-25433a6-dirty
Go Version: go1.14
Go OS/Arch: linux/amd64
operator-sdk Version: v0.15.2
Build Commit: 25433a6-dirty
Build Time: 2020-03-19T08:42:19Z
```

We should have a way to simply print `v0.0.2-next-25433a6-dirty`

```
╰─ ike version -s|--short
v0.0.2-next-25433a6-dirty
```

##### Why

It could be handy for scripting purposes.

### PR DESCRIPTION
**Pull request by bartoszmajsak**
#### Short description of what this resolves:

Provides `-s|--short` flag for `ike version`

Closes #390

### CODE REVIEW

File: pkg/cmd/version/version.go [hunk 1]
```diff
@@ -15,16 +15,31 @@ var log = logf.Log.WithName("version")
...
+			}
 			return nil
 		},
 	}
+
+	versionCmd.Flags().BoolP("short", "s", false, "prints only version without build details")
+	return versionCmd
+}
+
+func logShortVersion() {
```
Comment (aslakknutsen) [inline pkg/cmd/version/version.go]: Why is this private but the other public?

Comment (bartoszmajsak) [reply pkg/cmd/version/version.go]: b/c the other is called from the operator side (`ike serve`), so not only used by the command itself.

Comment (aslakknutsen) [reply pkg/cmd/version/version.go]: bob bob bob

Comment (bartoszmajsak) [reply pkg/cmd/version/version.go]: I know... but not sure anything else would be less bob bob bob

**Pull request merged**